Applications/Spire/Include/Spire/KeyBindings/KeyBindings.hpp

```diff
@@ -227,11 +290,39 @@ namespace Details {
       std::vector<KeyBindingMapping> m_bindings;
   };
 
+  template<typename T>
+  KeyBindings::Tag::Tag(int key, T value)
+    : Tag(key, boost::optional<T>(std::move(value))) {}
```

Comment:
Whenever possible, use deduction guides, so ```boost::optional(std::move(value))``` is fine.

Reply:
It won't compile, which I think is due to the fact that `boost::optional` supports references.

Reply:
Gotcha, that sucks.